ceph.spec.in

```diff
@@ -895,6 +895,7 @@ cmake .. \
     -DWITH_EMBEDDED=OFF \
     -DWITH_MANPAGE=ON \
     -DWITH_PYTHON3=ON \
+    -DWITH_MGR_DASHBOARD_V2_FRONTEND=OFF \
```

Comment:
Do we need to add something similar to the DEB builds?

Reply:
Yes

Reply:
fixed